pkg/utils/eden.go

```diff
@@ -0,0 +1,116 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	adamContainerName = "eden_adam"
+	adamContainerRef  = "lfedge/adam"
+)
+
+//StartAdam function run adam in docker with mounted adamPath/run:/adam/run
+//if adamForce is set, it recreates container
+func StartAdam(adamPort string, adamPath string, adamForce bool) (err error) {
+	if adamForce {
+		_, _, err = RunCommandAndWait("docker", strings.Fields(fmt.Sprintf("rm -f %s", adamContainerName))...)
```

Comment:
I am tempted to suggest that we should be using an official docker API Go client here instead of shelling out. What do you think @deitch ? I mean this one: https://docs.docker.com/engine/api/sdk/examples/

Reply:
Yeah, it probably is easier. Whatever you do, do _not_ use github.com/docker/docker, but rather the subsidiary repos, in this case github.com/docker/cli most likely.

I also would add that there is some duplication in the `if` and the `else`. Can we simplify that?

Reply:
Thanks for the comment, implemented using the SDK.